packages/phoenix-evals/src/phoenix/evals/models/openai.py

```diff
@@ -279,12 +279,33 @@ def _init_rate_limiter(self) -> None:
         )
 
     def _build_messages(
-        self, prompt: MultimodalPrompt, system_instruction: Optional[str] = None
+        self, prompt: MultimodalPrompt, audio_format: str = None, system_instruction: Optional[str] = None
```

Comment:
why do we need to pass an additional system_instruction here? a previous message part can contain that